src/main/java/edu/hawaii/its/api/controller/GroupingsRestControllerv2_1.java

```diff
@@ -660,4 +668,11 @@ public ResponseEntity getAsyncJobResult(@RequestHeader(CURRENT_USER_KEY) String
                 .ok()
                 .body(asyncJobsManager.getJobResult(currentUser, jobId));
     }
+    @GetMapping(value = "/announcements/active")
+    public ResponseEntity<List<String>> outageMessage() {
```

Comment:
Can we return the Announcements class here instead of List<String>? We are in process of making all API endpoints return a resultCode. 

As for the name of this endpoint, let's call it activeAnnouncements to match the URL. Don't forget to update to logger message too. We should also have the method name in the announcementsService also be called activeAnnouncements.

Reply:
Ok good point for matching the endpoint name to the URL, thank you.